Wangscape/tilegen/alpha/CalculatorTopTwo.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <utility>
+#include <vector>
+
+#include "CalculatorBase.h"
+
+
+namespace tilegen
+{
+namespace alpha
+{
+
+
+class CalculatorTopTwo :
+    public tilegen::alpha::CalculatorBase
+{
+public:
+    double power = 1.;
+protected:
+    virtual void updateAlphasAux(const Weights& weights);
```

Comment:
Hmm... at first I've missed a fact that's it's virtual :D. Yes, it's safe now, but `CalculatorBase` doesn't look like a typical interface. That class is abstract now, but it has a few non-pure virtual functions, so I think it (its "abstracness") may change some day. With `override` keyword I would feel a bit safier. It's already marked `virtual`, which does less than `override`, cause it's just a hint for programmer. `override` will generate an error during compilation. For me it's a rule of thumb to use `override` every time it's possible.